core/tests/tests/Http.mint

```diff
@@ -84,6 +84,19 @@ suite "Http.formDataBody" {
   }
 }
 
+suite "Http.jsonBody" {
```

Comment:
I'd like to see 2 more test cases:
- testing that the existing headers remain intact
- testing that the `Content-Type` record remains intact if set before-hand

Reply:
> testing that the existing headers remain intact

Have added this to an existing test. See [`81cba8b` (#434)](https://github.com/mint-lang/mint/pull/434/commits/81cba8bcdbb825209b6f4bea59b5f0702977937f)

> testing that the Content-Type record remains intact if set before-hand

Created a new test for this. See [`6a7af43` (#434)](https://github.com/mint-lang/mint/pull/434/commits/6a7af43fa7c0539ceca90eb1bf6bebdfbb9c9298)

Reply:
Great! 🙏